azure-kusto-data/tests/test_helpers.py

```diff
@@ -18,19 +18,32 @@ def test_dataframe_from_result_table():
         data = response_file.read()
 
     response = KustoResponseDataSetV2(json.loads(data))
-    df = dataframe_from_result_table(response.primary_results[0])
+    test_dict1 = {"RecordName": lambda col, frame: frame[col].astype("str"), "RecordInt64": lambda col, frame: frame[col].astype("int64")}
+    test_dict2 = {"int": lambda col, frame: frame[col].astype("int32")}
```

Comment:
Not sure about this test - doesn't int already get converted to int32? try a completely different type

Reply:
I wanted to test that the conversion of int types to int32 won't override the conversion by column name to int64. I'll add another test that converts to a completely different type.